test_runner/src/test/kotlin/ftl/util/LogTableBuilderTest.kt

```diff
@@ -93,7 +93,7 @@ internal class LogTableBuilderTest {
         val table = buildTable(*sampleColumns)
 
         // then
-        val tableLines = table.split(System.lineSeparator())
+        val tableLines = table.split("\n")
```

Comment:
Isn't old implementation os agnostic? I probably miss something here, but why this change?

Reply:
Windows System.lineseperator = \n\r
All the tests cases make use of \n and quite alot have simply copy pasted text with \n insertions for new lines.
This is includes yaml files and other files that were originally edited on mac/linux. Therefore for almost ALL cases where System.lineseperator was used under windows. It failed to either split or compare etc. Hence why the simple change of \n fixes the majority of issues.

Reply:
Thanks for explanation!

Reply:
@Sloox @pawelpasterz maybe we should create some variable with line end to always use it instead of `\n` or `System.lineSeparator()`?

Reply:
You can, but IMO it's not really it is not necessary.